### DISCUSSION TITLE
【灌水】不懂就问

### DISCUSSION THREAD

**Discussion by wushanef** [share121/inter-knot#2951] [其他]
这里有蜂群吗<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
![1724002108299](https://github.com/user-attachments/assets/a193ff61-7f5c-4382-b3bb-0c009807a2fd)

**Comment by Buer-Nahida** [comment]
还真有

**Comment by Yizakl** [comment]
如果你想要群聊我们没有
但我们有Discord服务器
（门槛较高）
